handbook/company/customer-first.md

```diff
@@ -1,48 +1,49 @@
+Currently, CEs are responsible for answering all how-to questions and handling all feature requests (whether these come up pre- or post-sales). Similarly, CS is responsible for troubleshooting something not working correctly (whether these come up pre- or post-sales), taking into account the prioritization considerations as provided by CEs, and (effective 2021-08-01) adhering to our contractual [SLAs](https://about.sourcegraph.com/handbook/support#slas) post-sales and 1 hour internal SLA response time pre-sales between 2am-5pm PT Monday-Friday (this will expand to 24/5 support as the team grows over the course of FY22) -- between now and 2021-08-01, as the CS team hires and onboards additional folks, they will adhere to a 24 hour SLA for all pre-sales issues between 9am-5pm PT Monday-Friday. As CS continues to hire during FYQ2 and new folks are onboarded, CS will also take on responsibility for how-to questions and feature requests for non-strategic customers.
 
 A few notes for CE:
 
 
-* If CE doesn’t know the answer to a how-to question, they ask for help in the ask-product-eng Slack channel. This channel is monitored by product, engineering, and CS and is a best effort approach (ie, there is no SLA for when someone might expect an answer).
-* Any other request for help from engineering should go through CS (for troubleshooting) or product (for feature requests) and not to engineering directly (for example, CEs are welcome to start troubleshooting and issue, but if they get stuck, the next step is to engage CS and CS will take responsibility for resolution and customer communication from there)
+* If CE doesn’t know the answer to a how-to question, they ask for help in the ask-product-eng Slack channel. This channel is monitored by product, engineering, and CS. It is a best effort approach (i.e., there is no SLA for response time for answers).
+* Any other request for help from engineering should go through CS (for troubleshooting) or product (for feature requests) and not to engineering directly. (For example, CEs are welcome to start troubleshooting an issue, but if they get stuck, the next step is to engage CS, and CS will take responsibility for resolution and customer communication from there)
 * CE routes all feature requests to product for consideration and prioritization
-* If a CE feels compelled to let a customer know CS will help them, they should do so without using loaded terms like “ASAP” or that sign-up CS for something they may not be able to honor
-	* *A member of our support team will help you with this. They typically are able to start digging in within a few hours at most, so you are in good hands! If anything has changed on your side to make this more urgent, let me know and I can be sure they are aware, too.*
```

Comment:
@virginiaulrich lmk what you think of the revised wording and process. My intent is to change the message from "Hey, I'm passing you off to support and they'll get back to you in a few hours while you wait" to "Hey, I'm still with you, but I'm going to loop in support in the next few hours and spend my time bringing them up to speed so they can bring this to resolution as quickly as possible." Whatever the SLA is, I think it's better to help the customer understand that the wait time doesn't mean they've been "dropped" until the next person comes along and picks them up. WDYT?